test/functionalTest/cases/1259_new_json_for_v2/update_attribute_with_null_value.test

```diff
@@ -34,13 +34,13 @@ brokerStart CB 0
 # 02. Get E1/A1 (see that A1 has null value)
 # 03. Create E2/A2='03'
 # 04. Modify E2/A2=null (PATCH)
-# 05. Get E2/A2 (A2 stays the same)
+# 05. Get E2/A2 (value of A2 has been replaced)
 # 06. Create E3/A3='06'
 # 07. Modify E3/A3=null (PUT)
 # 08. Get E3/A3 (value of A3 has been replaced)
 # 09. Create E4/A4='09'
 # 10. Modify E4/A4=null (POST)
-# 11. Get E4/A4 (A4 stays the same)
+# 11. Get E4/A4 (value of A4 has been replaced)
 # 12. Create (POST /v2/entities) E5/A5=null using URI param keyValues
```

Comment:
This here gives me the impression a bug fix has been done ...
Or is it just a fix of an erroneous functest of an older bugfix?